scipy/sparse/tests/test_base.py

```diff
@@ -3942,6 +3942,25 @@ def test_ticket1160(self):
         b[:,0] = 0
         assert_(len(b.keys()) == 0, "Unexpected entries in keys")
 
+    def test_copy(self):
```

Comment:
We already have a test for this: https://github.com/scipy/scipy/blob/master/scipy/sparse/tests/test_base.py#L1784